.travis.yml

```diff
@@ -50,7 +50,7 @@ before_install:
   - "travis_retry pip install twine wheel coveralls requirements-builder"
   - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
   - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder --level=dev --req requirements-devel-cernopendata.txt setup.py > .travis-devel-requirements.txt"
```

Comment:
We should then add separate matrix entry for Invenio devel packages to detect early failures.